frontend/src/components/hooks/useNotificationUsers.js

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react'
+import { getAllNotificationUser } from '../../service/notificationUserService'
+
+export default function useNotificationUsers() {
+  const [notificationUser, setNotificationUser] = useState([])
+
+  useEffect(() => {
+    getAllNotificationUser().then(res => {
```

Comment:
abbreviated variables make it harder to read #cleancode -> better use result or response than only res